Pin base repo commit for consistency

lucasaafaria opened the pull request:
Issue reference: #20 

Description: pins commit https://github.com/0xB10C/peer-observer/commit/87823b767c74e60b23c9a150984c7001d245277b when cloning `peer-observer` repo

m4ycon commented on docker/bitcoin-node.dockerfile:
```diff
@@ -48,6 +49,7 @@ RUN git clone -b $PEER_EXTRACTOR_BRANCH --single-branch $PEER_EXTRACTOR_REPO /pe
 
 # Set working directory to the repository
 WORKDIR /peer-observer
+RUN git checkout $PEER_EXTRACTOR_COMMIT
```
I think this command should be right after `git clone`, they are pretty related. Just to be organized, ok 👍 ? And I think its a good moment to update the commit hash, to a more recent one.

Also, after I merged another PR of yours, this branchs needs to be updated with `master`.

lucasaafaria replied:
Actually, the checkout must happen after the `WORKDIR` command. That's because when executed before, `git checkout` does not have the context of a git repo, and thus fail.

lucasaafaria commented:
> I think there was some mistake here in the update with master, I expect to see these commits: `old one -> merge from ClubeBitcoinUnB:master -> something new`. Can you fix this? So that git doesn't see these already merged commits as different commits from your branch?
> 
> Its quite simple, you can update your repo master (in github interface if you want), then you can undo locally some commits until [2aaf0a9](https://github.com/ClubeBitcoinUnB/peer-observer-docker/pull/23/commits/2aaf0a9ee0ed2e896612437b462d32da147f3a4b), merge master on your branch and re-commit [178e7d6](https://github.com/ClubeBitcoinUnB/peer-observer-docker/pull/23/commits/178e7d6e858b077c47797eec3a0e824520fd61d3) + [2adb7ea](https://github.com/ClubeBitcoinUnB/peer-observer-docker/pull/23/commits/2adb7ea5ab683130e9916d153898383920c1ad6a). And of course, to do this you have to force push as it will change git history here.

Fixed it, thanks. I updated my fork with the upstream, synced master with my branch and commited on top of it, so now history won't be messed up.
